src/features/goblins/merchant/MerchantModal.tsx

```diff
@@ -0,0 +1,271 @@
+import React, { useContext } from "react";
+import { useActor, useMachine } from "@xstate/react";
+import { Modal } from "react-bootstrap";
+
+import { Panel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+import * as AuthProvider from "features/auth/lib/Provider";
+import { Context } from "features/game/GoblinProvider";
+import { frogMachine } from "./lib/frogMachine";
+import { ErrorMessage } from "features/auth/ErrorMessage";
+
+import { ErrorCode } from "lib/errors";
+
+// images
+import frog_unrevealed from "../../../assets/nfts/frogs/frog_unrevealed.gif";
```

Comment:
We can use absolute paths for `assets`.

E.g. `assets/nfts/frogs/frog_unrevealed.gif`

Reply:
Used absolute paths for this file and other files as well 😄